worlds/kh1/Options.py

```diff
@@ -117,9 +117,9 @@ class Goal(Choice):
     
     Sephiroth: Defeat Sephiroth
     Unknown: Defeat Unknown
-    Postcards: Turn in all 10 postcards in Traverse Town
+    Postcards: Turn in an amount of postcards in Traverse Town
```

Comment:
Both lines here should use "number" and not "amount"
```suggestion
    Postcards: Turn in a number of postcards in Traverse Town
```